Program.cs

```diff
@@ -0,0 +1,405 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+using VisualCode.Entities;
+
+namespace VisualCode
+{
+    class Program
+    {
+        enum enumMenuMain : int {Maintenance = 1, RegistrySales = 2, Reports = 3};
+        enum enumMenuMaintenance : int {Customers = 1, Products = 2, Sellers = 3, MenuPrincipal = 4};
+        enum enumMenuReport : int {Sales = 1, CustomersAge = 2, CustomersBirthdayMonth = 3, ProductsStock = 4};
+
+        static List<Customers> customersList = new List<Customers>();

```

Comment:
variables must to have scope, private or public, etc, also if there are private must to start with underscore as prefix, more details in the coding standards document.